vmslave-config/pipeline/vmslave_config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+git diff --dirstat=files ${GIT_PREVIOUS_COMMIT} | grep "vmslave-config/"
```

Comment:
GIT_PREVIOUS_COMMIT is the previous commit found by jenkins git plugin.
If there's code change under vmslave-config, all slave will be automatic re-config.